neo/Helper.cs

```diff
@@ -181,6 +172,11 @@ public static uint ToTimestamp(this DateTime time)
             return (uint)(time.ToUniversalTime() - unixEpoch).TotalSeconds;
         }
 
+        public static ulong ToTimestampMS(this DateTime time)
+        {
+            return (ulong)(time.ToUniversalTime() - unixEpoch.ToUniversalTime()).TotalMilliseconds;
```

Comment:
unixEpoch must be in UTC now

```suggestion
            return (ulong)(time.ToUniversalTime() - unixEpoch).TotalMilliseconds;
```

Reply:
I did not understand this change, but makes sense because the function above is like this.
Can you explain?